javascript/exorelay/src/exorelay.ls

```diff
@@ -11,7 +11,7 @@ class ExoRelay extends EventEmitter
 
   (@config) ->
     @config?.exocom-host or throw new Error 'ExoCom host not provided to Exorelay'
-    @config?.exocom-port or throw new Error 'ExoCom port not provided to Exorelay'
+    @config.exocom-port? or @config.exocom-port = 80
```

Comment:
Thoughts on update this function to start with
```
(@config = {}) ->
```
Then we can remove the `?` from the exocom-host and role as well.

Reply:
I believe we can simplify this line to  `@config.exocom-port or= 80`